pretyping/evarsolve.ml

```diff
@@ -1554,19 +1554,17 @@ let solve_candidates unify flags env evd (evk,argsv) rhs =
       | l, [] -> evd
 
 let occur_evar_upto_types sigma n c =
-  let c = EConstr.Unsafe.to_constr c in
   let seen = ref Evar.Set.empty in
   (* FIXME: Is that supposed to be evar-insensitive? *)
```

Comment:
I guess now we are evar-insensitive on purpose?

Reply:
Yes, the comment should go away. I'll remove it with a force-push.

Reply:
Done.